gmaps/directions.py

```diff
@@ -48,6 +48,15 @@ class Directions(widgets.Widget):
         90 degrees north). Longitudes are expressed as a float
         between -180 (corresponding to 180 degrees west) and 180
         (corresponding to 180 degrees east).
+    :param avoid_ferries If true, instructs the Distance Matrix service to avoid 
+        ferries where possible.
```

Comment:
I think adding which GoogleMaps service is affected by parameters will confuse most users -- how about just "avoid ferries where possible"?

Also, you're missing a colon after the parameter name:

```
:param avoid_ferries: avoid ferries where possible (default: False)
```

(the same is true for the other parameters here)